requirements.txt

```diff
@@ -18,7 +19,7 @@ testtools==1.8.0  # python-subunit
 pbr>=0.11
 testscenarios==0.5.0  # python-subunit
 python-subunit==1.1.0  # lettuce
-requirements/src/lettuce-0.2.21.ccnmtl.tar.gz
+lettuce==0.2.21.ccnmtl
```

Comment:
Externals are configured to pull from pypi.python.org. The vendored lettuce build & django-courseaffils 2.0.6 need to be available there before we can pull these libraries from the source tree.